indico/htdocs/sass/widgets/_permissions.scss

```diff
@@ -95,10 +95,21 @@
                     margin: 0.4em 0.5em;
                     padding: 0.6em 1em;
                     text-transform: uppercase;
+                    color: $light-gray;
 
                     &.disabled {
                         @include opacity(0.5);
                     }
+
+                    &.visited {
```

Comment:
Is that really a good name? It doesn't really have anything to do with what it's used for...

Reply:
I wasn't sure about the name and I chose it because we used `.visited` in other places for that color. Do you think it'd be better to use the actual name of the permissions, i.e `.paper-manager`?

Reply:
Yep, maybe we could automatically add a `permission-<objtype.lower>-<name>` class to these elements? So you'd have e.g. `.permission-event-paper_manager` for the paper manager permission.